science/pymol/Portfile

```diff
@@ -83,7 +101,7 @@ use_parallel_build yes
 
 build {}
 
-destroot.cmd ${python.bin} setup.py --no-user-cfg --osx-frameworks --glut --use-msgpackc=no
```

Comment:
there is no reason to remove the `build` phase and change the `destroot` phase. The defaults work just fine, it will build in the `build` phase and just copy the files in the `destroot` phase.